skyplane/broadcast/bc_client.py

```diff
@@ -0,0 +1,117 @@
+import uuid
+from datetime import datetime
+from pathlib import Path
+
+from typing import TYPE_CHECKING, Optional, List
+
+from skyplane.api.client import tmp_log_dir
+from skyplane.api.usage.client import get_clientid
+from skyplane.api.dataplane import Dataplane
+from skyplane.broadcast import BroadcastDataplane
+from skyplane.broadcast.bc_planner import BroadcastDirectPlanner, BroadcastMDSTPlanner, BroadcastHSTPlanner, BroadcastILPSolverPlanner
+from skyplane.api.impl.provisioner import Provisioner
+from skyplane.api.transfer_config import TransferConfig
+from skyplane.utils import logger
+
+if TYPE_CHECKING:
```

Comment:
I didn't know about the `TYPE_CHECKING` constant! pretty useful to get around circular deps.